src/installer-and-application-po-files/kolibri-application.po

```diff
@@ -94,7 +94,7 @@ msgstr ""
 msgid "Installation in progress."
 msgstr ""
 
-msgid "Kolibri failed to start. Additional information can be found in the log file:"
+msgid "Kolibri failed to start. Additional information can be found in the log directory:"
```

Comment:
Maybe remove the `log` and put a space after the color so it reads: `... found in the directory:<space>`?